src/components/Search/Search.tsx

```diff
@@ -10,10 +10,22 @@ import Meta from '../Meta'
 import { canSearchForProperty } from '@/utils/userPermissions'
 import { PropertyListItem } from '@/models/propertyListItem'
 
-const Search = ({ query }) => {
+interface SearchProps {
+  query?: {
+    searchText?: string
+    pageNumber?: string
+  }
+}
+
+interface PropertyData {
+  total: string
```

Comment:
So long as `total` as string instead of a number doesn't cause any issues , then it's fine.